port_range/__init__.py

```diff
@@ -119,6 +119,13 @@ def parse(self, port_range):
 
         return port_from, port_to
 
+    def __eq__(self, other):
+        """ Compare two port ranges. """
+        return self.bounds == other.bounds
+
+    def __hash__(self):
+        return self.__str__().__hash__()
```

Comment:
Since self.bounds returns a tuple of ints, you could have used the hash value of the tuple directly as the following code shows:
```python
In [10]: class A:
    ...:     def __init__(self, a, b):
    ...:         self.a = a
    ...:         self.b = b
    ...:     @property
    ...:     def bounds(self):
    ...:         return self.a, self.b

In [16]: a = A(1, 2)
In [17]: b = A(1, 2)
In [22]: hash(a.bounds) == hash(b.bounds)
Out[20]: True
In [21]: b = A(1, 3)
In [22]: hash(a.bounds) == hash(b.bounds)
Out[22]: False
```